amgut/db/patches/0041.sql

```diff
@@ -15,13 +15,21 @@ CREATE TABLE pm.shotgun_adapter_aliquot (
  ) ;
 
  CREATE TABLE pm.shotgun_index_aliquot (
- 	shotgun_index_aliquot_id     bigserial  NOT NULL,
- 	name                         varchar  NOT NULL,
- 	notes                        varchar  ,
- 	limit_freeze_thaw_cycles     bigint  NOT NULL,
- 	CONSTRAINT pk_shotgun_index_aliquot PRIMARY KEY ( shotgun_index_aliquot_id ),
- 	CONSTRAINT idx_shotgun_index_aliquot UNIQUE ( name )
-  ) ;
+	shotgun_index_aliquot_id bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	notes                varchar  ,
+	limit_freeze_thaw_cycles bigint  NOT NULL,
+	CONSTRAINT pk_index_aliquot PRIMARY KEY ( shotgun_index_aliquot_id ),
+	CONSTRAINT idx_index_aliquot UNIQUE ( name )
+ );
+ 
+ CREATE TABLE pm.shotgun_index_tech (
+	shotgun_index_tech_id bigserial  NOT NULL,
+	name                 varchar(100)  ,
```

Comment:
Honestly, cause that was the default in dbSchema and didn't catch it. However, this is just the name of the tech, which in general is like 20 chars ...